frontend/src/static/js/v_summary.js

```diff
@@ -523,7 +539,12 @@ window.vSummary = {
         filtered.push(authorMap[author]);
       });
 
-      filtered.sort(window.comparator((ele) => ele[0][this.sortingOption]));
+      filtered.sort(window.comparator((ele) => {
```

Comment:
In both cases, filtered is a 2D array right? So why not you do something simpler like 
```
filtered.sort(window.comparator((group) => group.reduce(function(total, author) { 
  return total + author.sortingOption;
}));
```

Make it a side function so that it is neater and reusable for both cases.